collaboration-concept.md

```diff
@@ -0,0 +1,137 @@
+# Collaboration Concept
+
```

Comment:
I would start with a short motivational and overview, e.g.

"Collaboration is an often asked for feature in Theia. In this document we analyze the collaboration feature of VS Code, specify technical requirements and architecture for Theia and give an overview over the API concept."